dists/make-release.sh

```diff
@@ -25,12 +21,6 @@ fi
 bin_dir="bin/$VERSION"
 mkdir -p $bin_dir
 
-read -p "Which Rollbar token should be used in this release: " ROLLBAR_TOKEN
-if [[ -z $ROLLBAR_TOKEN ]]; then
-  echo "Rollbar token is mandatory" >&2
-  exit 2
-fi
```

Comment:
Rollbar is actually never used:

https://github.com/Scalingo/cli/blob/81aeb22b58168fc08865e8786ebdd64aced3c395/cmd/error.go#L41-L57

And I'm not sure we want to use it at some point...